proposals/lip-0045.md

```diff
@@ -189,7 +233,7 @@ The outbox root substore holds the root of the outbox of each partner chain.
 ##### Store Prefix, Store Key, and Store Value
```

Comment:
> The outbox root substore holds the root of the outbox of each partner chain.

I find this description misleading. It is not the outbox root of each partner chain, but rather the outbox root with respect to a partner chain, i.e., the root of the messages target that chain.

Reply:
This point is still not addressed.

Reply:
edited

Reply:
Looks good.